x/liquidstaking/spec/02_state.md

```diff
@@ -90,15 +104,16 @@ This will be consumed at **Handle Queued Withdraw Insurance Requests** when Epoc
 
 - reward module account’s native token(e.g. `acanto`) balance
 - sum of all chunk balance
-    - The chunk balance will only be as much as the balance accumulated from delegation rewards between epochs. At the end of each epoch, the cumulated chunk balance will be transferred to the reward module account.
+    - The chunk balance will only be as much as the balance accumulated from delegation rewards between epochs. 
+      At the end of each epoch, the cumulated chunk balance will be transferred to the reward module account.
     - When insurance is withdrawn and there are no candidate insurances, the chunk balance can be the same as the chunk size in tokens.
 - sum of all tokens corresponding delegation shares of paired chunks
     - total amount of native tokens currently delegated
     - may be less than the sum of the delegation shares due to slashing in the calculation
         - This will be an edge case because insurance will cover any penalty.
 - sum of all remaining rewards of all chunks delegations
     - not yet claimed native tokens
-        - `cumulated delegation rewards x (1 - paired insurance commission rates)`
+        - `cumulated delegation rewards x (1 - paired insurance commission rates - module's fee rate)`
```

Comment:
This part doesn't match the code, we need to discuss which is right, and `CalcNetAmount` and `CalcConservativeNetAmount` coexist and are used differently for each case, so we need to check if this is the right direction, and if it is right, the content should be added to the spec, or it would be better to unify it into one NetAmount

Reply:
@dongsam 
Thanks for the review. I'll review NetAmount related parts with `Taeyoung` and update the spec.
I think re-defining CalcNetAmount's logic as `CalcConservativeNetAmount` would be ok.
Then the meaning of NetAmount in lsm will be the same and there will be less confusions.

Reply:
Added 5.75% description also in this [commit](https://github.com/b-harvest/Canto/pull/18/commits/71fcb97c89d8d443ae09fbb86d3f3f8bde05534f)

Reply:
Removed ConservativeNetAmount and update calc method of TotalLiquidTokens (consider paired insurance's balance).

commit link: https://github.com/b-harvest/Canto/pull/18/commits/d6ed3cf8946f2d1b4b355e87a9ec0b9494bb0a2a